index.rst

```diff
@@ -100,73 +110,268 @@ Data Sets Types and Goals
+WIDE_GAMA15H    9615    HSC-Y   26        380^384^388^404^408^424^426^436^440^442^446^452^456^458^462^464^468^470^472^474^478^27032^27034^27042^27066^27068
+WIDE_GAMA15H    9615    TOTAL   112       Size: 1.4 TB
+============    =====   ======  ========= ==========
+
+=========   =====   ======  ========= ==========
+Field       Tract   Filter  NumVisits Visit List
+=========   =====   ======  ========= ==========
+UD_COSMOS   9813    HSC-G   17        11690^11692^11694^11696^11698^11700^11702^11704^11706^11708^11710^11712^29324^29326^29336^29340^29350
+UD_COSMOS   9813    HSC-R   16        1202^1204^1206^1208^1210^1212^1214^1216^1218^1220^23692^23694^23704^23706^23716^23718
+UD_COSMOS   9813    HSC-I   33        1228^1230^1232^1238^1240^1242^1244^1246^1248^19658^19660^19662^19680^19682^19684^19694^19696^19698^19708^19710^19712^30482^30484^30486^30488^30490^30492^30494^30496^30498^30500^30502^30504
+UD_COSMOS   9813    HSC-Z   31        1166^1168^1170^1172^1174^1176^1178^1180^1182^1184^1186^1188^1190^1192^1194^17900^17902^17904^17906^17908^17926^17928^17930^17932^17934^17944^17946^17948^17950^17952^17962
+UD_COSMOS   9813    HSC-Y   52        318^322^324^326^328^330^332^344^346^348^350^352^354^356^358^360^362^1868^1870^1872^1874^1876^1880^1882^11718^11720^11722^11724^11726^11728^11730^11732^11734^11736^11738^11740^22602^22604^22606^22608^22626^22628^22630^22632^22642^22644^22646^22648^22658^22660^22662^22664
+UD_COSMOS   9813    NB0921  28        23038^23040^23042^23044^23046^23048^23050^23052^23054^23056^23594^23596^23598^23600^23602^23604^23606^24298^24300^24302^24304^24306^24308^24310^25810^25812^25814^25816
+UD_COSMOS   9813    TOTAL   177       Size: 3.2 TB
+=========   =====   ======  ========= ==========
+
+   This dataset satisfies the definition above for a MEDIUM dataset.
+
+LARGE
+=====
+
+1. HSC SSP PDR1 and PDR2
+
+  The full HSC SSP Public Data Release 1 (PDR1) dataset has been processed by LSST DM twice.  This is a LARGE dataset.  The timescale for these runs is essentially as-needed.  The processing of these large dataset could be increased as the workflow and orchestration tooling for automated execution improves.  We expect this scale of processing to always require some manual supervision (but significantly less than it does today).  As more data becomes available with future SSP public releases, we expect this dataset to grow to include them.
+
+  See reports at:
+
+    * `Cycle S17 HSC PDR1 Processing <https://confluence.lsstcorp.org/display/DM/S17B+HSC+PDR1+reprocessing>`_
+    * `Cycle S18 HSC PDR1 Processing <https://confluence.lsstcorp.org/display/DM/S18+HSC+PDR1+reprocessing/>`_
+
+  The HSC Public Data Release 2 (PDR2) dataset was released by HSC in the Summer of 2019.  This dataset has been copied to NCSA and is now available at `/datasets/hsc/raw/ssp_pdr2`.  It is appropriate for DRP and for AP testing and performance monitoring.  As with PDR1, PDR2 is similarly a LARGE dataset.
```

Comment:
Note: the PDR2 dataset has not yet been transferred to NCSA (that directory is empty!).  Paul is sorting out some issues with transfer speeds with the Japanese, but I expect it will land soon-ish (given the big demand for it).

Again, is it worth specifying the on-disk sizes here?  Hsin-Fang would know those numbers (surprisingly, I couldn't glean the total #s easily from her extensive write-ups on confluence & JIRA). 

If useful, PDR1 includes:
-5654 visits in 7 bands (grizy + NB0921 + NB
- 3 depth layers (WIDE, DEEP, UDEEP)
- covers 119 tracts
- total node[core] hours: 9227[79246] (of which up to 80% can be spent in *multiband.py*)
- input dataset is ~13x larger than RC2